ocs_ci/utility/utils.py

```diff
@@ -1736,6 +1737,7 @@ def ceph_health_check_base(namespace=None):
         boolean: True if HEALTH_OK
 
     """
+    return True
```

Comment:
This shouldn't be here.

Reply:
Great catch, that could be _really_ bad... It's a local workaround that shouldn't've been pushed